ingestion-beam/src/test/java/com/mozilla/telemetry/contextualservices/ParseReportingUrlTest.java

```diff
@@ -165,7 +167,7 @@ public void testRequiredParamCheck() {
       return null;
     });
 
-    PAssert.that(result.output()).satisfies(messages -> {
+    PAssert.that(result.output().setCoder(getCoder())).satisfies(messages -> {
```

Comment:
why not use `SponsoredInteraction.getCoder()`?

Reply:
I thought I had found all the instances of that. 🤦  thank you!